resources/views/components/menu/account.blade.php

```diff
@@ -39,7 +39,7 @@
             <div class='softcore cursor-help' title="Points earned in softcore mode">{{ localized_number($user->RASoftcorePoints) }}</div>
         @endif
     </div>
-    <x-nav-dropdown trigger-class="py-0" dropdown-class="dropdown-menu-right">
+    <x-nav-dropdown trigger-class="py-0" dropdown-class="dropdown-menu-right" desktopHref="{{ route('user.show', $user) }}">
```

Comment:
can be `:desktopHref="route('user.show', $user)"` to condense it. no strong preference.

Reply:
Thanks, addressed in https://github.com/RetroAchievements/RAWeb/pull/1840/commits/c654adefaa3d2f62f630c9d280733fe78daa1975.